css/style.css

```diff
@@ -558,3 +558,284 @@ textarea.form-element {
+    width: 83px;
+    margin-right: 0;
+  }
+
+  .backdev-desktop {
+    display: flex;
+    width: 105px;
+    margin-right: 0;
+  }
+
+  .tonic-desktop {
+    display: flex;
+    line-height: 52px;
+    color: #091e42;
+    font-weight: 700;
+  }
+
+  .uber-width {
+    width: 44px;
+  }
+
+  .lead-width {
+    width: 115px;
+  }
+
+  .para-desktop {
+    display: flex;
+    color: #344563;
+    width: 93%;
+    align-items: flex-start;
+  }
+
+  .htm-desktop {
+    display: flex;
+    width: 86px;
+  }
+
+  .grid-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 80%;
+    padding: 0%;
+    margin-left: 10%;
+    grid-template-areas:
+      "item1 item2"
+      "item4 item3"
+      "item5 item6"
+      "item8 item7";
+  }
+
+  .grid-item1 {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 1rem;
+    grid-area: item1;
+    margin-bottom: 5rem;
+  }
+
+  .grid-item2 {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+    border-top-right-radius: 1rem;
+    grid-area: item2;
+    margin-bottom: 5rem;
+  }
+
+  .grid-item4 {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    border-top-left-radius: 1rem;
+    grid-area: item4;
+    margin-bottom: 5rem;
+  }
+
+  .grid-item3 {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 1rem;
+    grid-area: item3;
+    margin-bottom: 5rem;
+  }
+
+  .grid-item5 {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 1rem;
+    border-top-left-radius: 1rem;
+    grid-area: item5;
+    margin-bottom: 5rem;
+  }
+
+  .grid-item6 {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+    border-top-right-radius: 1rem;
+    grid-area: item6;
+    margin-bottom: 5rem;
+  }
+
+  .grid-item7 {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 1rem;
+    grid-area: item7;
+    margin-bottom: 5rem;
+  }
+
+  .grid-item8 {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    border-top-left-radius: 1rem;
+    grid-area: item8;
+    margin-bottom: 5rem;
+  }
+
+  .headline-top {
+    padding-left: 200px;
+  }
+
+  .flex-container-desktop {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .flex-desktop-item {
+    width: 50%;
+  }
+
+  .flex-desktop-item2 {
+    padding-top: 158px;
+  }
+
+  .lang {
+    margin-bottom: 60px;
+  }
+
+  .fram-des {
+    margin-bottom: 0;
+  }
+
+  .support-text-headline-about {
+    width: 600px;
+    padding-bottom: 30px;
+  }
+
+  .but-about {
+    width: 170px;
+    height: 40px;
+    border: 0;
+    flex-basis: 5%;
+    border-radius: 6px;
+  }
+
+  .about-self {
+    width: 140px;
+  }
+
+  .pro {
+    width: 200px;
+    display: block;
+  }
+
+  .flex-container-lang {
+    display: flex;
+    width: 60%;
+  }
+
+  .flex-container-lang-item {
+    width: 122px;
+  }
```

Comment:
kindly fix these design issues in your about me section to match the Figma design : 

- [ ] kindly align the language's title with the small arrow in front of it 
- [ ] kindly increase the width of the technologies squares (javascript, HTML, CSS) , align the content to the left, and make sure that they have the same gap between them as shown in the Figma design
- [ ] kindly add the lines below frameworks & skills items 🙏 

Your Design         | Figma Design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/200125046-a709fc2c-7b94-435c-8f4b-0f3289224e6b.png) | ![image](https://user-images.githubusercontent.com/89282221/200125149-d2931178-6d99-4094-baed-3a5296aedaa8.png)

Reply:
Well done for implementing the changes requested by the previous reviewer on the project cards. Kindly implement the part regarding the languages section as quoted below:


> kindly fix these design issues in your about me section to match the Figma design :
> 
> * [ ]  kindly align the language's title with the small arrow in front of it
> * [ ]  kindly increase the width of the technologies squares (javascript, HTML, CSS) , align the content to the left, and make sure that they have the same gap between them as shown in the Figma design
> * [ ]  kindly add the lines below frameworks & skills items 🙏
> 
> Your Design	Figma Design
> ![image](https://user-images.githubusercontent.com/89282221/200125046-a709fc2c-7b94-435c-8f4b-0f3289224e6b.png)	![image](https://user-images.githubusercontent.com/89282221/200125149-d2931178-6d99-4094-baed-3a5296aedaa8.png)